README.md

```diff
@@ -15,240 +15,7 @@ The Graph: https://thegraph.com/docs/en/
-  feesEth: BigDecimal
-  tokens: [Token!]!
-}
-
-HourlyTotal --o Token : tokens
-
-class TokenDailyTotal {
-  id: ID!
-  token: Token!
-  timestamp: BigInt!
-  totalVolume: BigInt!
-  totalVolumeUsd: BigDecimal!
-  totalVolumeEth: BigDecimal!
-  totalTrades: BigInt!
-  openPrice: BigDecimal!
-  closePrice: BigDecimal!
-  higherPrice: BigDecimal!
-  lowerPrice: BigDecimal!
-  averagePrice: BigDecimal!
-}
-
-class TokenHourlyTotal {
-  id: ID!
-  token: Token!
-  timestamp: BigInt!
-  totalVolume: BigInt!
-  totalVolumeUsd: BigDecimal!
-  totalVolumeEth: BigDecimal!
-  totalTrades: BigInt!
-  openPrice: BigDecimal!
-  closePrice: BigDecimal!
-  higherPrice: BigDecimal!
-  lowerPrice: BigDecimal!
-  averagePrice: BigDecimal!
-}
-
-class TokenTradingEvent {
-  id: ID!
-  token: Token!
-  trade: Trade!
-  timestamp: BigInt!
-  amountEth: BigDecimal!
-  amountUsd: BigDecimal!
-}
-
-TokenTradingEvent --o Token : token
-
-class Pair {
-  id: ID!
-  token0: Token!
-  token1: Token!
-  volumeToken0: BigInt
-  volumeToken1: BigInt
-  volumeTradedEth: BigDecimal
-  volumeTradedUsd: BigDecimal
-}
-
-Pair --o Token : token0 
-Pair --o Token : token1 
-
-class PairDaily {
-  id: ID!
-  token0: Token!
-  token1: Token!
-  timestamp: BigInt
-  volumeToken0: BigInt
-  volumeToken1: BigInt
-  volumeTradedEth: BigDecimal
-  volumeTradedUsd: BigDecimal
-}
-
-PairDaily --o Token : token0 
-PairDaily --o Token : token1 
-
-class PairHourly {
-  id: ID!
-  token0: Token!
-  token1: Token!
-  timestamp: BigInt
-  volumeToken0: BigInt
-  volumeToken1: BigInt
-  volumeTradedEth: BigDecimal
-  volumeTradedUsd: BigDecimal
-}
-
-PairHourly --o Token : token0 
-PairHourly --o Token : token1 
-```
-
-### Settlement entity notes
-
-So far settlements are being created using txHash as ID and that hash is creating the relation between Settlement and Trade entities.
-
-### Orders entity notes
-
-Notice order entity has 3 different timestamps. Each timestamp will be filled depending on the entry point that order passed.
-
-**Entrypoints are:**
-
-- On trade event
-- On invalidate order event
-- On pre sign event
-
-### About token prices
-
-In token entity you can find prices for each token. It's price it's being calculated using Honeyswap pools prices in gnosis chain or UniswapV3 in ethereum mainnet or rinkeby. 
-
-### About totals
-
-Totals is a singleton entity where it's aggregated information about how many different tokens had been traded, total traded volume in usd and eth, etc. You can find more detailed information in the subgraph schema. Please notice there's also a totals calculated by hour and by day.
-
-### About pairs
-
-Pairs are entities for a pair of tokens that had been traded on our platform. For keeping the tokens ordered in the same way, addresses are being sorted and token0 always stores the lower value and token1 the greater one. Please notice there's also a pairs entity by hour and by day.
+## There's further information about the model [here](./model.md)
```

Comment:
Just don't create the link in the title. 

also simplify title: Model should be good enough

Reply:
Done